### ISSUE TITLE
libcoral on Windows

### ISSUE THREAD

**Issue by abucka** [google-coral/libcoral#15]
Hello,

How can I build libcoral do Windows?

**Comment by manoj7410** [comment]
Have you tried to use a Virtual Box on Windows ?

**Comment by abucka** [comment]
no, I was just using Windows for building libcoral, which scripts should I use exactly or how to build it with Bazel?

**Comment by manoj7410** [comment]
I haven't tried to build the libcoral on Windows. Only followed [these](https://github.com/google-coral/libcoral#docker) instructions on Linux. You can try to run Ubuntu on Virtual Box and then try the compilation.

**Comment by abucka** [comment]
I ran docker_build.bat and got .exe files in the out directory, how can I generate the .lib files? My bazel-out folder is empty

**Comment by hjonnala** [comment]
@abucka can you please just try the [build.bat](https://github.com/google-coral/libcoral/blob/master/scripts/windows/build.bat) from libcoral repo.

**Comment by hjonnala** [comment]
@abucka can you please share the .exe files in the out directory. Thanks!

**Comment by abucka** [comment]
This is my out directory:
[out.zip](https://github.com/google-coral/libcoral/files/7692381/out.zip)

**Comment by colarocker** [comment]
So does anybody know how to build the libraries for windows / ms visual studio? Tried it for some days now but it's not really well documented. could i use docker for building the x64 libraries and use them in windows?

**Comment by ArchitB22976** [comment]
I'd second @colarocker here. I'm also trying to build this on a Win10 machine with no success.
